### DISCUSSION TITLE
HyperHDR v17 - Philips Hue Wizard - Save button isn't working

### DISCUSSION THREAD

**Discussion by sofakng** [awawa-dev/HyperHDR#120] [General]
I'm trying to create a second instance using my Philips Hue ceiling bulbs.  I'm following the Quick Start guide but on the screen where I select my Group ID and the Position/State for my lights, the save button doesn't do anything:

![image](https://user-images.githubusercontent.com/2228499/133824916-ecb307ef-3b62-47a2-a9f2-825e37df1c33.png)

I'm using Chrome v93.0.4577.82 (Windows).

**Comment by awawa-dev** [comment]
Start from fresh configuration or simply reset your HyperHDR Philips hue configuration for that instance (set different lamp type for example debug->file, restart HyperHDR and configure Philips again) because your configuration is broken after the import or after switching on to off 'API entertaiment mode' (or reverse) or rerunning the wizard (authentication part, not lamp definition as changing their configuration is safe).

**Comment by sofakng** [comment]
Thanks.  It's working now.

Also, I can create a new thread/topic but is there a guide on reducing latency?

I'm using HyperESP32 + HyperHDR + MS2109 capture and the latency is really good, but I feel it's about a half-second delayed.  It's definitely not terrible but I'm wondering how close to 'instant' I can have it?

**Comment by awawa-dev** [comment]
First: check your video processing latency. It's in logs, logged every minute. if it's < 10 ms then your must investigate the problem outside the application (video signal path: splitters/AVR etc. , slow LED drivers like Philips Hue) or start looking for a faster grabber (Ezcap 320/321/322).
That subject was recently mention as off-topic here: https://github.com/awawa-dev/HyperHDR/issues/118#issuecomment-920771835

**Reply by sofakng** [reply]
Thanks for the information.  It looks like my setup is around 8ms on average.

Would a faster grabber (Ezcap) still make a difference?

(Also, I have a question about adding relay support to HyperESP32;  Would you prefer I continue asking questions in this thread, or create separate threads?)

**Reply by awawa-dev** [reply]
Try to disable smoothing to exclude that module misconfiguration. New models from Ezcap are advertised as having 50ms latency, MS2109 is somewhere between 200-300ms. For HyperESP32  better let's go to it's github site ;)
